### PR TITLE
feat(): Add Soul Snad

### PR DESCRIPTION
**Pull request by RebelliousUno**
Adds Soul Snad,  currently using the regular MC soul sand texture to be updated later.

Adds Configuration for speed upgrades for Snad and Soul Snad
Soul Snad set to 4 rather than 2 by default

### CODE REVIEW

File: src/main/java/net/darkmorford/jas/configuration/ConfigurationData.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package net.darkmorford.jas.configuration;
+
+public class ConfigurationData {
+	public static final int SNAD_SPEED_INCREASE_DEFAULT = 2;
+	public static final String SNAD_SPEED_INCREASE_KEY = "Snad Speed increase multilpier";
```
Comment (DarkMorford) [inline src/main/java/net/darkmorford/jas/configuration/ConfigurationData.java]: "multiplier" is misspelled in this string, and the capitalization is inconsistent with the string for Soul Snad.

**Review by DarkMorford — CHANGES_REQUESTED** [review]
Other than the string issue I marked, this looks pretty good. Based on McJty's configuration tutorial, though, I'd like to add a check during `postInit` to see if the config has changed and save it if so. He suggests that configuration changing during the initialization process is not uncommon, so we should catch that if/when it happens.

File: src/main/java/net/darkmorford/jas/configuration/ConfigurationData.java (hunk 1 above)
Comment (RebelliousUno) [reply src/main/java/net/darkmorford/jas/configuration/ConfigurationData.java]: Whoops fixing

**Comment by RebelliousUno** [comment]
I've added the postInit check though we've also got an event `onConfigurationChanged` which I presume would catch it anyways?
Or maybe not...

**Review by DarkMorford — APPROVED** [review]

**Comment by DarkMorford** [comment]
Thanks for updating that. I agree that the `onConfigurationChanged` event *should* catch it, but with this codebase you never know. ;)

**Pull request merged**